hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -3244,18 +3244,21 @@ private void prepareDeleteTimestamps(Mutation mutation, Map<byte[], List<Cell>>
 
   private void updateDeleteLatestVersionTimestamp(Cell cell, Get get, int count, byte[] byteNow)
       throws IOException {
-    List<Cell> result = get(get, false);
+    try (RegionScanner scanner = getScanner(new Scan(get))) {
+      List<Cell> result = new ArrayList<>();
```

Comment:
Same here. I think it needs a comment to prevent a developer 'optimizing' replacing w/ a get.

Reply:
Yes, I'll add comments, thanks.